draft-ietf-quic-recovery.md

```diff
@@ -450,11 +458,10 @@ this packet adds network load without establishing packet loss.
 ## Generating Acknowledgements
 
 QUIC SHOULD delay sending acknowledgements in response to packets, but MUST NOT
-excessively delay acknowledgements of packets containing frames other than ACK.
-Specifically, implementations MUST attempt to enforce a maximum
-ack delay to avoid causing the peer spurious timeouts.  The maximum ack delay
-is communicated in the `max_ack_delay` transport parameter and the default
-value is 25ms.
+excessively delay acknowledgements of retransmittable packets. Specifically,
+implementations MUST attempt to enforce a maximum ack delay to avoid causing
+the peer spurious timeouts.  The maximum ack delay is communicated in the
+`max_ack_delay` transport parameter and the default value is 25ms.
 
```

Comment:
Just a stray thought: would some kind of hysteresis be of relevance? E.g. receiver should allow for 10% overrun of ack delay? Or perhaps this is already taken into account.

Reply:
The current model is the receiver communicates it's max ack delay.  The receiver is in the best place to determine how late alarms are likely to fire, so I think they should include that in the value they send.  That being said, it's probably worth stating that specifically.  Possibly in the transport draft?

Reply:
Makes sense

Reply:
PR #1960 merged already fyi.

Reply:
Yes I noticed, looks good, thanks.